packages/babel-core/src/transformation/plugin-pass.ts

```diff
@@ -7,19 +7,32 @@ export default class PluginPass<Options = object> {
   file: File;
   opts: Partial<Options>;
 
-  // The working directory that Babel's programmatic options are loaded
-  // relative to.
+  /**
+   * The working directory that Babel's programmatic options are loaded
+   * relative to.
+   */
   cwd: string;
 
-  // The absolute path of the file being compiled.
+  /** The absolute path of the file being compiled. */
   filename: string | void;
```

Comment:
With `/**` style of comment it's a jsdoc description which could be used as quick documentation in IDE.